crates/core-subsystem/core-resolver/src/validation/arguments_validator.rs

```diff
@@ -201,6 +208,9 @@ impl<'a> ArgumentValidator<'a> {
         number: &Number,
         pos: Pos,
     ) -> Result<Val, ValidationError> {
+        // println!("{:?}", argument_definition);
+        // returns InputValueDefinition { description: None, name: Positioned { pos: Pos(0:0), node: Name("limit") }, ty: Positioned { pos: Pos(0:0), node: Type { base: Named(Name("Int")), nullable: false } }, default_value: None, directives: [] }
+
```

Comment:
I was hoping to get the `@range` min and max values from `.exo` schema and somehow compare it with the provided value but I'm having trouble accessing the schema from my `index.exo`

Reply:
That was the motivation behind https://github.com/exograph/exograph/commit/7d88c2cbd83411e41032009e67bac5b3cce8eb42. The first step was to replace `async_graphql` types with our own. The second step would have been to enhance those replacements with more information (range, min/max, etc.). The final step would have been to use those enhancements to perform deeper validations (the TODO listed in https://github.com/exograph/exograph/blob/main/crates/core-subsystem/core-resolver/src/validation/arguments_validator.rs#L50, but also the ones you are implementing).

Let me know what you think of that as a prerequisite (perhaps a separate PR).